dict/portuguese.yml

```diff
@@ -2,290 +2,290 @@
-    'BT': [Butão, Reino do Butão, Butanês*, Butanense*, Butani*, Butâni*, Timbu, Thimphu, Timpu, Thimbu]
-    'IN': [Índia, República da Índia, Indian*, Bombaim, Mumbai, Nova Deli, Nova Delhi, Nova Déli]
-    'IR': [Irã, Irão, República Islâmica do Irã, República Islâmica do Irão, Iranian*, Teerã, Teerão]
-    'LK': [Sri Lanka, Sri Lanca, Seri Lanca, República Democrática Socialista do Sri Lanka, Ceilão, Cingalês*, Srilankês*, Srilanquês*, Colombo, Columbo]
-    'MV': [Maldivas, República das Maldivas, Maldiv*, Maldívi*]
-    'NP': [Nepal, República Democrática Federal do Nepal, Nepalês, Nepales*, Catmandu]
-    'PK': [Paquistão, República Islâmica do Paquistão, Paquistanês*, Islamabade, Islamabad]
+    'BT': [Butão, Butanês*, Butanense*, Butani*, Butâni*, Timbu, Thimphu, Timpu, Thimbu]
+    'IN': [Índia, Indian*, Bombaim, Mumbai, Nova Deli, Nova Delhi, Nova Déli]
+    'IR': [Irã, Irão, Iranian*, Teerã, Teerão]
+    'LK': [Sri Lanka, Sri Lanca, Seri Lanca, Ceilão, Cingalês*, Srilankês*, Srilanquês*, Colombo, Columbo]
+    'MV': [Maldivas, Maldiv*, Maldívi*]
+    'NP': [Nepal, Nepalês, Nepales*, Catmandu]
+    'PK': [Paquistão, Paquistanês*, Islamabade, Islamabad]
 
   SOUTH EAST:
     'BN': [Brunei, Estado de Brunei Darussalam, Estado de Brunei Darussalã, Brunean*, Bruneín*]
-    'ID': [Indonésia, República da Indonésia, Indonési*, Jacarta]
-    'KH': [Camboja, Cambodja, Reino do Camboja, Cambojan*, Phnom Penh, Pnom Pen, Pnom Pene, Penome Pene]
-    'LA': [Laos, República Democrática Popular do Laos, Laosian*, Laocian*, Vientiane, Vienciana]
-    'MM': [Myanmar, Birmânia, República da União de Myanmar, Mianmarense*, Birmanês*, Birmã*, Birman*, Birmanian*, Birmanense*, Rangum, Yangon, Nepiedó, Naypyidaw, Nay Pyi Taw]
+    'ID': [Indonésia, Indonési*, Jacarta]
+    'KH': [Camboja, Cambodja, Cambojan*, Phnom Penh, Pnom Pen, Pnom Pene, Penome Pene]
+    'LA': [Laos, Laosian*, Laocian*, Vientiane, Vienciana]
+    'MM': [Myanmar, Birmânia, Mianmarense*, Birmanês*, Birmã*, Birman*, Birmanian*, Birmanense*, Rangum, Yangon, Nepiedó, Naypyidaw, Nay Pyi Taw]
     'MY': [Malásia, Malaio*, Malaia*, Malási*, Kuala Lumpur, Cuala Lumpur, Quala Lumpur, Cualalampur, Putrajaia, Putrajaya]
-    'PH': [Filipinas, República das Filipinas, Filipin*,  Manila]
+    'PH': [Filipinas, Filipin*,  Manila]
     'SG': [Singapura, Cingapura, Singapurense*, Singapuran*]
-    'TH': [Tailândia, Reino da Tailândia, Tailandês*, Banguecoque, Bangkok, Bancoque]
-    'TL': [Timor-EAST, República Democrática de Timor-EAST, Timorense*, Maubere*, Díli]
-    'VN': [Vietnã, Vietname, República Socialista do Vietname, República Socialista do Vietnã, Vietnamit*, Vietnamês*, Vietnamense*, Vietnamian*, Hanói, Cidade de Ho Chi Minh, Saigon, Saigão]
+    'TH': [Tailândia, Tailandês*, Banguecoque, Bangkok, Bancoque]
+    'TL': [Timor-Leste, Timorense*, Maubere*, Díli]
+    'VN': [Vietnã, Vietname, Vietnamit*, Vietnamês*, Vietnamense*, Vietnamian*, Hanói, Cidade de Ho Chi Minh, Saigon, Saigão]
```

Comment:
VN is a difficult case. In principle, we don't include "Saigon" as it is old name for Ho Chi Minh City. Do people call it "Saigon" today?

Reply:
I am not sure how to proceed here. I might be mistaken, but I would say that it is not uncommon for people in general, at least in Brazil, to recognize the city more as Saigon than by it's current name (we even have a bossa nova song, launched in the 1980's - that became fairly popular and it is still known - called Saigon using the Vietnam war to build a metaphor).

I am open to your decision on whether to keep it or not.

Reply:
Other dictionary maintenors agreed with you on https://github.com/koheiw/newsmap/issues/47". Let's keep "Saigon" and change "Ho Chi Minh Citiy" to "Ho Chi Minh".